sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/services/OrchestrationService.java

```diff
@@ -0,0 +1,268 @@
+package com.sap.ai.sdk.app.services;
+
+import static com.sap.ai.sdk.app.controllers.OpenAiController.send;
+import static com.sap.ai.sdk.orchestration.OrchestrationAiModel.GPT_35_TURBO;
+import static com.sap.ai.sdk.orchestration.OrchestrationAiModel.Parameter.TEMPERATURE;
+
+import com.sap.ai.sdk.core.AiCoreService;
+import com.sap.ai.sdk.orchestration.AzureContentFilter;
+import com.sap.ai.sdk.orchestration.AzureFilterThreshold;
+import com.sap.ai.sdk.orchestration.DpiMasking;
+import com.sap.ai.sdk.orchestration.Message;
+import com.sap.ai.sdk.orchestration.OrchestrationChatResponse;
+import com.sap.ai.sdk.orchestration.OrchestrationClient;
+import com.sap.ai.sdk.orchestration.OrchestrationModuleConfig;
+import com.sap.ai.sdk.orchestration.OrchestrationPrompt;
+import com.sap.ai.sdk.orchestration.model.DPIEntities;
+import com.sap.ai.sdk.orchestration.model.DataRepositoryType;
+import com.sap.ai.sdk.orchestration.model.DocumentGroundingFilter;
+import com.sap.ai.sdk.orchestration.model.GroundingModuleConfig;
+import com.sap.ai.sdk.orchestration.model.GroundingModuleConfigConfig;
+import com.sap.ai.sdk.orchestration.model.Template;
+import com.sap.cloud.sdk.cloudplatform.thread.ThreadContextExecutors;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nonnull;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseBodyEmitter;
+
+/** Service class for the Orchestration service */
+@Service
+@Slf4j
+public class OrchestrationService {
+  private final OrchestrationClient client = new OrchestrationClient();
+
+  @Getter
+  private final OrchestrationModuleConfig config =
+      new OrchestrationModuleConfig().withLlmConfig(GPT_35_TURBO.withParam(TEMPERATURE, 0.0));
+
+  /**
+   * Chat request to OpenAI through the Orchestration service with a simple prompt.
+   *
+   * @return the assistant response object
+   */
+  @Nonnull
+  public OrchestrationChatResponse completion() {
+    final var prompt = new OrchestrationPrompt("Hello world! Why is this phrase so famous?");
```

Comment:
(Minor/Suggestion)

To motivate the Controller / Service distinction, I would move the String as part of the method input, e.g.

```suggestion
  public OrchestrationChatResponse completion( String famousPhrase) {
    final var prompt = new OrchestrationPrompt(famousPhrase+" Why is this phrase so famous?");
```